tests/native-lib/scalar_arguments.c

```diff
@@ -1,27 +1,35 @@
 #include <stdio.h>
 
-int add_one_int(int x) {
+// See comments in build_native_lib()
+#define EXPORT __attribute__((visibility("default")))
+
+EXPORT int add_one_int(int x) {
   return 2 + x;
 }
 
-void printer() {
+EXPORT void printer(void) {
```

Comment:
`void printer()` was causing a warning in clang, so added `(void)` here and below.

Reply:
Lol I entirely forgot that that's how C works...